filebeat/harvester/harvester.go

```diff
@@ -17,12 +17,15 @@ import (
 	"fmt"
 	"regexp"
 
+	"github.com/satori/go.uuid"
+
 	"github.com/elastic/beats/filebeat/config"
 	"github.com/elastic/beats/filebeat/harvester/encoding"
 	"github.com/elastic/beats/filebeat/input"
 )
 
 type Harvester struct {
+	Id                 uuid.UUID
```

Comment:
Why does a havester need a UUID? As far as I can tell, this value is transient, right?